### PR TITLE
Guideline for DLT Multinode demo

### PR DESCRIPTION
**Pull request by duvanan13**
Guideline for DLT Multinode demo:
+ Setup Docker
+ Setup network bridge
+ Fetch logs from Container to Host

### CODE REVIEW

File: Section_4_Demo_docker/test.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Hướng Dẫn Multinode Demo với Docker Setup
```
Comment (minminlittleshrimp) [inline Section_4_Demo_docker/test.md]: Please prepare the doc in English

File: Section_4_Demo_docker/test.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+ Multinode Demo với Docker Setup tại [đây](https://inside-docupedia.bosch.com/confluence2/display/EASHome/Multinode+Demo+with+Docker+setup).
```
Comment (minminlittleshrimp) [inline Section_4_Demo_docker/test.md]: Why attaching an internal link here?
@duvanan13  please prepare your own doc to fit in with OSS community.

Comment (duvanan13) [reply Section_4_Demo_docker/test.md]: I'm sorry for being so subjective so I have update new changes below.
Thanks for remind me!!!
<3

File: Section_4_Demo_docker/test.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        "IPv4Address": "192.168.2.3/24",
+        "IPv6Address": ""
+    }
+},
+```
+
+## 2.7. Install dlt bin and lib in the container:
+
+```bash
+root@2ff461cda2f5:/home/nad8hc/work/docker/dlt-daemon# cp ./build/src/daemon/dlt-daemon /usr/local/bin/
```
Comment (minminlittleshrimp) [inline Section_4_Demo_docker/test.md]: replace nad8hc with <$USER>

**Review by minminlittleshrimp — APPROVED** [review]

**Pull request merged**